src/tic_tac_toe/announcer.clj

```diff
@@ -1,6 +1,6 @@
 (ns tic-tac-toe.announcer)
 
-(def human-move "Please pick a spot from 1-%d: ")
+(def human-move "Enter s to save game.\n\nPlease pick a spot from 1-%d: ")
```

Comment:
I'm not sure I like the wording here, I think I like an interface more like
```
please pick a spot from 1-9 (enter "s" to save and exit)
> 
```

Reply:
[Updated](https://github.com/saralein/tic-tac-toe_clojure/pull/9/files#diff-25c91f67a97e706c5a657b280e537c55R3).